STYLEGUIDE.md

```diff
@@ -72,3 +78,74 @@ import 'camera.dart';
 import 'game.dart';
 import 'viewport.dart';
 ```
+
+### Identifiers
+
+Use all [effective dart patterns](https://dart.dev/guides/language/effective-dart)
+
+* Avoid using global constants, but if you do use, name it prefixing with a `k`.
+
+```dart
+// Do not:
+const someConstant = 42;
+
+// Do instead:
+const kSomeConstant = 42;
+```
+
+### Comments
+
+* Add an identifier for TODO comments
+* Use full capitalized TODO
+
+```dart
+
+// Do not:
+// Todo: This thing should be that thing
+final thisThing = 13;
+
+// Do instead:
+// TODO(wofstein): This thing should be that thing
```

Comment:
```suggestion
// TODO(wolfenrain): This thing should be that thing
```
:laughing:

Reply:
I reffuse.

Reply:
If anything it should have been `wolfstein`.